### PR TITLE
Submission for CV Bootcamp - Zeyad Moustafa

### PR DESCRIPTION
**Pull request by ZeyadMoustafa01**
Hello team, this is my PR for the computer vision bootcamp in hopes to join the team and contribute to the work being done. Please take a look at the images and code below and comment any suggestions. Thanks!

image
loss
accuracy

### CODE REVIEW

**Review by mika-m-shaw — APPROVED** [review]
Everything looks well done. You could even add a function e.g train() that does the model.fit() to make the code even more object oriented, and then have a main() functions that calls everything. Besides that great job and welcome to the team :)
